packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -100,17 +100,22 @@ class Directionality extends InheritedWidget {
   /// the given context.
   ///
   /// If there is no [Directionality] ancestor widget in the tree at the given
-  /// context, then this will return null.
+  /// context, then this will throw an assertion error.
```

Comment:
Maybe mention that in debug mode it throws an assertion, and in release mode throws an exception.